src/main/java/com/untamedears/JukeAlert/listener/JukeAlertListener.java

```diff
@@ -6,11 +6,7 @@
 import static com.untamedears.JukeAlert.util.Utility.notifyGroup;
 
 import java.sql.SQLException;
-import java.util.Map;
```

Comment:
We're going to ping pong here.

Leave it listed out. Half of folks have their IDE set to list it out, the other half will consolidate; I'm of a mind to just say "do no harm" and just don't change imports unless adding a new one :).

It's not a big deal but if you can reverse this specific piece of change, that'd be great.

Reply:
Agree, wildcard imports are bad style, see https://stackoverflow.com/questions/147454/why-using-a-wild-card-with-a-java-import-statement-bad

Reply:
I also dislike those imports, my IDE just did it anyway and I only noticed it after making the request